### ISSUE TITLE
[salesforce] Provide as a salesforce app

### ISSUE THREAD

**Issue by ceolter** [ag-grid/ag-grid#633]

**Comment by ceolter** [comment]
tracked by https://ag-grid.atlassian.net/browse/AG-98

**Comment by makinggoodsoftware** [comment]
Hi, 

Thank you for raising this issue for us, at this moment we are reviewing how we process Feature Requests/questions that come through from GitHub.

As I am sure you are aware, we offer two products:

ag-Grid - this is entirely free to use in line with our MIT license
ag-Grid Enterprise - our commercial version which funds the development of both versions.

We currently have a very full pipeline of work which can be viewed here: https://www.ag-grid.com/ag-grid-pipeline/

This will be our priority for the coming months and as a result, we don’t have capacity to take feature requests/questions from GitHub, but you might find that yours has already been raised in our pipeline.

We will do a periodic review of GitHub issues for any bugs raised and these can also be tracked on our pipeline page.

Thanks for your understanding.
ag-Grid Team.

**Comment by sagarpomal** [comment]
Hi @makinggoodsoftware ,

**Comment by sjurgis** [comment]
@sagarpomal did you find any useful library or solution for grids/datatables in lwc?

**Comment by sagarpomal** [comment]
@sjurgis  No, we could not find any other solution related to datatable in lwc with features that ag-grid has. We tried our best to make ag-grid work in lwc by contacting ag-grid and also working with Salesforce. Based on our discussion, ag-grid has to make a lot of breaking changes, and they can't take this on priority because of low volume on Salesforce.

We ended up using ag-grid with Lightning Container based on our use case and features we wanted and validated our approach with Salesforce based on current limitations.

We could have tried to modify the lwc datatable; however, it is yet to be made open source under lwc base components due to its complexity.
